### ISSUE TITLE
Nested code blocks have problems when copying by Copy button

### ISSUE THREAD

**Issue by calmwaves111** [mugiwara85/CodeblockCustomizer#51]
`````
````
```dataviewjs
const files=dv.pages
```
````
`````

The copied content will become
```
<div><div class="codeblock-customizer-line-number"><span class="codeblock-customizer-line-number-element">1</span></div><div class="codeblock-customizer-line-text">```dataviewjs</div></div><div><div class="codeblock-customizer-line-number"><span class="codeblock-customizer-line-number-element">2</span></div><div class="codeblock-customizer-line-text">const files=dv.pages</div></div><div><div class="codeblock-customizer-line-number"><span class="codeblock-customizer-line-number-element">3</span></div><div class="codeblock-customizer-line-text">```</div></div>
```

**Comment by mugiwara85** [comment]
Ahh. I will look at this  today

**Comment by mugiwara85** [comment]
Sorry,  I can't reproduce this. Did this happen in edit mode or reading mode? If you can reproduce it multiple times, please describe how did you do it.

**Comment by mugiwara85** [comment]
Nevermind, I can reproduce it.

**Comment by mugiwara85** [comment]
Can you please check if the problem still persists in version 1.2.2?

**Comment by calmwaves111** [comment]
hello, I found that this is not a problem with nested code, but rather a problem with not specifying the code language
Even if I only wrote three points but did not specify a code language, this situation would occur
![image](https://github.com/mugiwara85/CodeblockCustomizer/assets/56907077/d8f07fa3-6f71-47c9-8e4a-d83971d63d07)
while I specify a code language, it work well
![image](https://github.com/mugiwara85/CodeblockCustomizer/assets/56907077/676137ed-ad00-4c8c-9004-ea9679c4717c)

**Comment by mugiwara85** [comment]
oh I see. This exaplins why I was able to reproduce it once, and why I wasn't it other time. I will look into this again. Thanks for the info!

**Comment by mugiwara85** [comment]
As it turned out, this was the case not only for nested code blocks, but for every code block, without a language specified. I fixed it in version 1.2.3. Please test it out, and if there is a problem report back. I am closing the issue. If there is some problem, I'll reopen it.

**Comment by calmwaves111** [comment]
Now it's working properly，thanks to your hard work.
